rules/005_microflows/005_0002_commit_actions_with_a_loop_test.rego

```diff
@@ -0,0 +1,112 @@
+package app.mendix.microflows.commit_actions_with_a_loop
+import rego.v1
+
+
+# Test data
+loop_commit_good = {
+  "$Type": "Microflow$Microflow",
+  "Name": "MicroflowForLoop",
+  "MainFunction": [
+    {
+      "Attributes": {
+        "$Type": "Microflows$LoopedActivity",
+        "Documentation": "",
+        "ErrorHandlingType": "Rollback",
+        "LoopSource": {
+          "$Type": "Microflows$IterableList",
+          "ListVariableName": "BikeList",
+          "VariableName": "IteratorBike"
+        },
+        "ObjectCollection": {
+          "$Type": "Microflows$MicroflowObjectCollection",
+          "Objects": [
+            {
+              "$ID": "REoQzbooxUqlJfxtM61uuw==",
+              "$Type": "Microflows$ActionActivity",
+              "Action": {
+                "$ID": "ePBJZv2ockmxBjt1gRJInw==",
+                "$Type": "Microflows$ChangeAction",
+                "ChangeVariableName": "IteratorBike",
+                "Commit": "No",
```

Comment:
if `Commit` is `Yes`; this should be a negative case. Can you add this? Also the test cases can be a lot shorter. The test cases now contain a lot of unnecessary attributes/data. It should be stripped down to the essentials to highlight what is being checked.